index.ts

```diff
@@ -369,7 +369,9 @@ async function listNeurons() {
 
   if (neurons.length > 0) {
     neurons.forEach((n) => {
-      log(`Neuron ID: ${n.neuronId}`);
+      if (n.votingPower > 0) {
```

Comment:
Just realized that voting power may be zero if the dissolve delay is under 6 months. So this condition should likely be different. Too lazy to figure out the correct condition at this hour, though.

Reply:
We have something similar in the nns-dapp.

I can apply it here. We check that the neuron stake + maturity is greater than the transaction fee.

I can add a simple check like that here if you want.

Reply:
That would be very useful, thanks.